src/index.scss

```diff
@@ -25,6 +25,11 @@ body {
   flex-direction: column;
 }
 
+address {
```

Comment:
Should this style be in TargetPage.scss file?

Reply:
I can scope it just for the Target-page.